thirdeye-ui/src/app/components/app-bar/app-header/app-header.styles.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2024 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import { makeStyles } from "@material-ui/core";
+import { AppHeaderStyleProps } from "./app-header.inerfaces";
+
+export const useStyles = (
+    props: AppHeaderStyleProps
+): ReturnType<typeof makeStyles> =>
```

Comment:
rename var to something specific to component. since we are exporting useStyles it can be accidentally imported in other projects

Reply:
Done